task/test-github

```diff
@@ -60,7 +60,11 @@ def mockServer():
             self.wfile.write(value.encode('utf-8'))
             self.wfile.flush()
 
-        def replyJson(self, value, headers={}, status=200):
+        def replyJson(self, value, headers=None, status=200):
+            if headers:
```

Comment:
I'm not a huge fan of the shadowing here.

In any case I feel like this is candidate for a one-liner

```
headers = headers.copy() if headers else {}
```

I also sort of prefer this pattern: `new=dict(old, key=val)` but because "Content-type" is not a valid kwarg name, we can't do it...

Maybe something like

```
our_headers = { "Content-type": "application/json" }

... self.replyData(... headers=dict(headers, **our_headers))
```
?

Or just inline the entire thing...

Reply:
Reworked as per IRC.